signer/src/main/java/com/uport/sdk/signer/UportHDSigner.kt

```diff
@@ -292,12 +292,16 @@ class UportHDSigner : UportSigner() {
         val prefs = context.getSharedPreferences(ETH_ENCRYPTED_STORAGE, MODE_PRIVATE)
         //list all stored keys, keep a list off what looks like uport root addresses
         return prefs.all.keys
+                .asSequence()
```

Comment:
Long chain of operations on lists is worth turning into sequences because then the operations are applied more efficiently.